app/operations/beta/project_board_operations.rb

```diff
@@ -39,10 +39,13 @@ def project_labels
       end
 
       def project
-        @project ||= @projects_scope
-          .friendly
-          .preload(:users, stories: %i[notes document_files tasks])
-          .find(@project_id)
+        return raise ActiveRecord::RecordNotFound if @current_user.blank?
```

Comment:
Can this be done in the first line of the `run` method instead of here? Also, we should avoid leaking ActiveRecord errors from operations, prefer to use a result object like we do here https://github.com/Codeminer42/cm42-central/blob/d1a22a77ea6e007be02d7b43a00bb949194e870c/app/operations/beta/project_board_operations.rb#L29